docker/qgisserver/geomapfish_qgisserver/accesscontrol.py

```diff
@@ -271,7 +271,7 @@ def get_roles(self):
 
         parameters = self.serverInterface().requestHandler().parameterMap()
 
-        if parameters.get("USER_ID") == "0":
+        if parameters.get("USER_ID") == "0" or parameters.get("USER_ID") == "2147483647":
```

Comment:
Seems not required as USER_ID=0 is passed in request parameters and should not require any role relationships.